tests/Cookie/CookieJarTest.php

```diff
@@ -37,6 +37,11 @@ public function testCreatesFromArray()
         $this->assertCount(2, $jar);
     }
 
+    public function testEmptyJarIsCountable()
+    {
+        self::assertCount(0, new CookieJar());
```

Comment:
Accepted `$this->`, will fix this. 

Did not understand about `instanceOf`. This test explicitly invokes the `count()` method to be sure, that internals of `CookieJar` is countable from construction with 7.2